Delicious.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="em">
+<head>
+    <meta charset="utf-8">
+    <title>Delicious</title>
+</head>
+<body>
+	<header>
+		<span><img src="logo.png" alt="logo"></span>
+		<h1>Delicious</h1>
+		<span>The BEST FOOD BLOG ON THE WEB</span>
+		<ul>
+			<span class=icon><a href="facebook"><img src="fb-icon.png" alt="facebook"></a></span>
+			<span class=icon><a href="twitter"><img src="twit-icon.png" alt="twitter"></a></span>
+			<span class=icon><a href="google plus"><img src="gp-icon.png" alt="google plus"></a></span>
+			<span class=icon><a href="instagram"><img src="insta-icon.png" alt="instagram"></a></span>
+			<span class=icon><a href="flickr"><img src="flic-icon.png" alt="flicker"></a></span>
+			<span class=icon><a href="pinterest"><img src="pint-icon.png" alt="pinterest"></a></span>
+		</ul>
+		<ul>
+			<span class=icon><a href="RSS"><img src="RSS-icon.png" alt="RSS"></a></span>
+			<span class=icon><a href="email"><img src="mail-icon.png" alt="email"></a>
+		</ul>
+	</header>
+	<div class="divider"></div>
+	<section>
+		<h2>Chocolate Pizza</h2>
+		<span>POSTED ON 15 DEC 2013/DESSERTS</span>
+		<span><a href="printerlink"><img src="print-icon.png" alt="printer icon">PRINT</a></span>
+		<img src="choco-pizza.png" alt="cheesecake with candy decorations">
+		<p class="recipe">For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. Add salt with one or two stirs, set aside and let cool completely.</p>
+		<p class="recipe">Ice cream: In a small pot over medium heat, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p>
+		<p class="recipe">Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. freeze for at least two hours before serving.</p>
+		<div class="recipecard">
+			<form>
+    		<table>
```

Comment:
While a table does appear to gain the effect you're looking for, our recipe list is just that: A list.